syz-ci/manager.go

```diff
@@ -946,6 +943,10 @@ func (mgr *Manager) uploadCoverJSONLToGCS(gcsClient gcs.Client, mgrSrc, gcsDest
 	return eg.Wait()
 }
 
+func dateTimeSuffix(dt time.Time) string {
+	return fmt.Sprintf("-%s-%d-%d", dt.Format(time.DateOnly), dt.Hour(), dt.Minute())
```

Comment:
You can use dt.Format("-2006-01-02-15-04") instead, this will spare you a function (you also won't need to test it).